CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 
 cmake_minimum_required(VERSION 2.8.12)
 
+if(NOT CMAKE_VERSION MATCHES "ReactOS")
```

Comment:
Did you test this?

Reply:
Yes: "cmake version 3.2.1-ReactOS" and "cmake version 3.11.0".